### PR TITLE
Copy/paste of targets into observations

### PR DESCRIPTION
**Pull request by toddburnside**

### CODE REVIEW

File: explore/src/main/scala/explore/tabs/TargetTabContents.scala [hunk 1]
```diff
@@ -717,16 +731,31 @@ object TargetTabContents extends TwoPanels:
                         .getOrElse(IO.unit)
                     else IO.unit
 
+                  case LocalClipboard.CopiedTargets(tids) =>
+                    (props.focused.obsSet, optViewAgwo).tupled
+                      .foldMap((obsIds, agwov) =>
+                        val undoContext    = UndoContext(props.listUndoStacks, agwov)
+                        // don't want to do anything if the target astersism already contains the targets
```
Comment (cquiroz) [inline explore/src/main/scala/explore/tabs/TargetTabContents.scala:738]: Do you mean we don't put the same target twice in the obs?

**Review by cquiroz — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/tabs/TargetTabContents.scala (hunk 1 above)
Comment (toddburnside) [reply explore/src/main/scala/explore/tabs/TargetTabContents.scala:738]: I know, crazy, right?

Comment (toddburnside) [reply explore/src/main/scala/explore/tabs/TargetTabContents.scala]: Your comment made me realize this bit needed improvement. 😄

**Pull request merged**